frontend/src/metabase/query_builder/components/ResponsiveParametersList.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useCallback, useState, useMemo } from "react";
+import useIsSmallScreen from "metabase/hooks/use-is-small-screen";
+import Button from "metabase/core/components/Button";
+
+import { Parameter } from "metabase-types/types/Parameter";
+
+import {
+  FilterButton,
+  ParametersListHeader,
+  StyledParametersList,
+  ResponsiveParametersListRoot,
+  ParametersListContainer,
+} from "./ResponsiveParametersList.styled";
+
+interface ResponsiveParametersListProps {
+  parameters: Parameter[];
+  setParameterValue: (parameterId: string, value: string) => void;
+  setParameterIndex: (parameterId: string, parameterIndex: number) => void;
+}
+
+export const ResponsiveParametersList = ({
+  parameters,
+  setParameterValue,
+  setParameterIndex,
+}: ResponsiveParametersListProps) => {
+  const [mobileShowParameterList, setShowMobileParameterList] = useState(false);
+  const isSmallScreen = useIsSmallScreen();
+
+  const handleFilterButtonClick = useCallback(() => {
+    setShowMobileParameterList(!mobileShowParameterList);
+  }, [mobileShowParameterList]);
+
+  const activeFilters = useMemo(() => {
+    return parameters.filter(p => !!p.value).length;
+  }, [parameters]);
+
+  return (
+    <ResponsiveParametersListRoot>
+      {isSmallScreen && (
+        <FilterButton
+          borderless
+          primary
+          icon="filter"
+          onClick={handleFilterButtonClick}
+        >
+          {activeFilters > 0 ? `${activeFilters} active filters` : `Filters`}
+        </FilterButton>
+      )}
+      <ParametersListContainer
+        isSmallScreen={isSmallScreen}
+        mobileShow={mobileShowParameterList}
```

Comment:
Do you think something along the line `isShowingMobile` would better describe this flag? There are also a lot of usages of `isShowingXxx` in our projects, so it seems to be a de-facto standard.
![image](https://user-images.githubusercontent.com/1937582/183601064-e74729f6-0dcd-49a1-81a0-8237569de306.png)

Reply:
Yeah, I'll update the name so that it looks more like out other properties